blueocean-pipeline-api-impl/src/test/resources/io/jenkins/blueocean/rest/impl/pipeline/earlyUnstableStatusShouldReportPunStateAsRunningAndResultAsUnknown.jenkinsfile

```diff
@@ -1,6 +1,6 @@
 node {
     stage ("stage 1 marked as unstable") {
-        currentBuild.result = 'UNSTABLE';
+        unstable('foo')
```

Comment:
We should look into whether `RunWrapper.setResult` can be deprecated. Tricky because this was done as a method on a POJO rather than a `Step`, so we have no `TaskListener` available. Ideally `currentBuild` would be deprecated as a whole: https://github.com/jenkinsci/workflow-basic-steps-plugin/pull/71